v2/pkg/cnirpc/mock_test.go

```diff
@@ -82,7 +83,7 @@ func TestCNIWithMock(t *testing.T) {
 	dialFunc := func(ctx context.Context, a string) (net.Conn, error) {
 		return dialer.DialContext(ctx, "unix", a)
 	}
-	conn, err := grpc.Dial(s.sockName, grpc.WithInsecure(), grpc.WithContextDialer(dialFunc))
+	conn, err := grpc.Dial(s.sockName, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithContextDialer(dialFunc))
```

Comment:
`grpc.WithInsecure()` is  is deprecated.
https://stackoverflow.com/questions/70482508/grpc-withinsecure-is-deprecated-use-insecure-newcredentials-instead
https://github.com/grpc/grpc-go/blob/v1.47.0/dialoptions.go#L278-L313